File: estrutura de dados/entrada de dados/questoes/questao07.c
#include <stdio.h>

// Questão 7: Cálculo de Imposto e Salário Final
void questao7() {
    float salario, imposto, salario_final;
    
    printf("Digite o salario: ");
    scanf("%f", &salario);
    
    imposto = salario * 0.10;
    salario_final = salario - imposto;
    
    printf("Salario: %.4f\n", salario);
    printf("Imposto: %.4f\n", imposto);
    printf("Salario Final: %.4f\n", salario_final);
}

int main() {
    questao7();
    
    return 0;
}

File: algoritmos/06 - Funções/Lista02/questoes/questao07.py
def calcular_contracheque():
    nome = input("Digite o nome do funcionário: ")
    salario = float(input("Digite o salário do funcionário: "))

    imposto_renda = salario * 0.15
    inss = salario * 0.10

    salario_final = salario - imposto_renda - inss

    print(f'''
          DIVINOS SOFTWARE CARIDADES
          FUNCIONÁRIO: {nome}
          DEMONSTRATIVO:
          Salário: R$ {salario:.2f}
          Imposto: R$ {imposto_renda:.2f}
          INSS: R$ {inss:.2f}
          Total a Receber: R$ {salario_final:.2f}''')


calcular_contracheque()
